src/main/java/at/gammastrahlung/monopoly_server/network/websocket/MonopolyMessageHandler.java

```diff
@@ -223,4 +224,55 @@ private static ServerMessage endCurrentPlayerTurn(WebSocketPlayer player) {
 
         return initiateRound(player);
     }
+
+    private static ServerMessage handleStartAuction(ClientMessage clientMessage, Game game) {
+        Property property = game.getGameBoard().getPropertyById(Integer.parseInt(clientMessage.getMessage()));
+        if (property == null) {
+            return ServerMessage.builder()
+                    .messagePath("error")
+                    .jsonData("Property not found.")
+                    .type(ServerMessage.MessageType.ERROR)
+                    .build();
+        }
+        game.startAuction(property);
+        return ServerMessage.builder()
+                .messagePath("auction_start")
+                .jsonData("Auction started for " + property.getName())
+                .type(ServerMessage.MessageType.SUCCESS)
+                .build();
+    }
+
+    private static ServerMessage handlePlaceBid(ClientMessage clientMessage, Game game) {
+        if (game.getCurrentAuction() == null) {
+            return ServerMessage.builder()
+                    .messagePath("error")
```

Comment:
Error is not a valid message path that can be handled by the client.
Message path should be the same for all message possibilities of an action.